dev-tools.sh

```diff
@@ -89,7 +89,7 @@ elif [ "$1" == "test" ]; then
     MIGRATION_MODE=$3
     if [ ! ${MIGRATION_MODE} ]; then
         if [ "${TEST_MODE?}" == "normal" ]; then
-            MIGRATION_MODE="single-db"
+            MIGRATION_MODE="multi-db"
```

Comment:
Please update line 36 with default migration mode as ```multi-db```.

Reply:
Also, update the comment here as well: https://github.com/babelfish-for-postgresql/babelfish_extensions/blob/BABEL_4_X_DEV/INSTALLING.md.tmpl#L247 along with updating the following query there